### DISCUSSION TITLE
Accessing the raw user call to OpenAI

### DISCUSSION THREAD

**Discussion by grand-cat-unknown** [567-labs/instructor#210] [Q&A] [answered]
Hello community!

Is there a way to access the raw user message that is sent to OpenAI ?
I want to basically see if my Field descriptions of my desired Class output are being used or not.

for example, for this class:

class DesiredClass(BaseModel):
    lineId: str = Field(..., description="The xml-path to determine the line id")

I want to know if the (description="The xml-path to determine the line id" ) is being used in the OpenAI call or not.

Thanks in advance!

**Comment by ydennisy** [comment]
+1 on this request, it would be great to have an option to pass a flag to print the call being made.

**Comment by jxnl** [comment, marked as answer]
yes it is, if you want to see what is send to openai via the json schema just run  `DesiredClass.model_json_schema()` and you'll see the whole payload.

**Reply by grand-cat-unknown** [reply]
Thanks!
